src/templates/index.html

```diff
@@ -37,13 +37,21 @@
     <div ng-controller="map-ctrl">
       <div class="section-one">
         <div class="row">
-	  <div class="col-md-12">
+	  <div class="col-md-8">
             <div style="height: 800px;" id="map">
             </div>
           </div>
+	  <div class="col-md-4">
```

Comment:
Please move **Social media tweets** implementation from this branch. It will create merge conflict.